loggins.sh

```diff
@@ -5,31 +5,40 @@
 autoload -U add-zsh-hook
 
 # Start derby
+source load.zsh &> /dev/null
+
+#iset +m
+derby &> /dev/null
+sleep 1
 echo "Welcome to Valet 2000"
-echo "Loading command database..."
-#./derby.sh &> /dev/null
-echo "Valet is online"
+
+#echo "Loading command database..."
+#echo "Valet is online"
 
```

Comment:
I also think I don't like the init messages in general. Output is more useful for user initiated actions.